File: JavaWeb2-angular/src/app/model/user/UserListRequest.ts
import {CommonConstant} from '../../constant/CommonConstant';

export class UserListRequest {

  userName:string = CommonConstant.EMPTY;//用户名

  personName:string = CommonConstant.EMPTY;//用户姓名

  roleName:string = CommonConstant.EMPTY;//角色名

  createStartDate:any = null;//用户创建的开始日期

  createEndDate:any = null;//用户创建的结束日期

  currentPage = 1;//当前页数

  pageSize = 10;//每页显示多少条

}

File: JavaWeb2-eureka-client-user/src/main/java/com/javaweb/web/eo/user/UserListRequest.java
package com.javaweb.web.eo.user;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UserListRequest implements Serializable {
	
	private static final long serialVersionUID = -3616575788300683386L;

	private String userName;//用户名
	
	private String personName;//用户姓名

    private String roleName;//角色名
	
	private String createStartDate;//用户创建的开始日期
	
	private String createEndDate;//用户创建的结束日期
	
	private Long currentPage = 1L;//默认当前第1页
	
	private Long pageSize = 10L;//默认每页显示10条

}
